### ISSUE TITLE
Usability improvements: key bindings

### ISSUE THREAD

**Issue by pdav** [pimutils/khal#462]
1- there should be two clear sets of key bindings: one when the focus is in the calendar browser (left column) and one when the focus is in the event list (right column)
2- if an action cause the focus to change, it should be easily reverted: at this time, if I press the left arrow when in the event list, the focus is set to the calendar browser. The right arrow cannot bring me back to the event list and perform a totally different action (move to another day).

Suggestions:

- Create two distinct subsections in `[keybindings]`
- Use only tab (or another key) to switch between calendar/event list.
- Allow esc to return to calendar (as with vi). Prohibit left and right arrows in event list.

```
[keybindings]
  [[calendar]]
  focuseventlist: tab
  new: ...
  mark: ...
  # no `view` nor `duplicate` allowed in this subsection

  [[eventlist]]
  focuscalendar: tab, esc
  save: ...
  new: ...
  # no `left` nor `right` allowed in this subsection
```

**Comment by geier** [comment]
No, this isn't possible yet. The whole area of keybindings is suboptimal at the moment. Also see #462.
